### PR TITLE
Minor language corrections and improvements

### PR DESCRIPTION
**Pull request by tomchadwin**
Complete only up to "Plugins" heading (l. 322) - more to follow

### CODE REVIEW

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 1]
```diff
@@ -50,20 +50,20 @@ Menu Bar
...
+shortcuts are the default settings, but they can be reconfigured using
 the :guilabel:`Keyboard Shortcuts` via the :menuselection:`Settings -->` menu.
 
-Most Menu options have a corresponding tool and vice-versa, however, the Menus
-are not organized exactly like the toolbars. The location of menu options
-in the toolbars are indicated below in the table. Plugins may load in additional
-menu options to Menus. For more information about tools and toolbars, see section 
+Most Menu options have a corresponding tool and vice-versa. However, the Menus
+are not organized exactly like the toolbars. The locations of menu options
+in the toolbars are indicated below in the table. Plugins add new
```
Comment (havatv) [inline source/docs/user_manual/introduction/qgis_gui.rst]:
```suggestion
in the toolbars are indicated below in the table. Plugins may add new
```

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 2]
```diff
@@ -75,13 +75,13 @@ The :menuselection:`Project` menu provides access and exit points of
...
+  a template (see :ref:`Project files options <projectfiles_options>` for
   template configuration)
-* :guilabel:`Open...` a project file from either a file browser or PostgreSQL database;
+* :guilabel:`Open...` a project file from either a file browser or PostgreSQL database
 * :guilabel:`Close` a project or revert it to its last saved state
-* :guilabel:`Save` a project in a :file:`.qgs` or :file:`.qgz` file format, either as file
+* :guilabel:`Save` a project in :file:`.qgs` or :file:`.qgz` file format, either as a file
   or within a PostgreSQL database
-* Export the project rendering to different formats from the map canvas or
+* Export the project as an image file to different formats from the map canvas or
```
[the remark this answers is not in the document]
Comment (tomchadwin) [reply source/docs/user_manual/introduction/qgis_gui.rst]: Disagree re: SVG, but 100% agree re: PDF. Thanks. Will accept, but feel its still a bit verbose, with project, map, and canvas all mentioned. Will take another look.

[the remark this answers is not in the document]
Comment (tomchadwin) [reply source/docs/user_manual/introduction/qgis_gui.rst]:
```suggestion
* Export the map canvas to different formats or
```

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 3]
```diff
@@ -75,13 +75,13 @@ The :menuselection:`Project` menu provides access and exit points of
...
   using a :ref:`print layout <label_printlayout>`
```
Comment (tomchadwin) [inline source/docs/user_manual/introduction/qgis_gui.rst]:
```suggestion
  use a :ref:`print layout <label_printlayout>` for more complex output
```

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 4]
```diff
@@ -75,13 +75,13 @@ The :menuselection:`Project` menu provides access and exit points of
...
+  a template (see :ref:`Project files options <projectfiles_options>` for
   template configuration)
-* :guilabel:`Open...` a project file from either a file browser or PostgreSQL database;
+* :guilabel:`Open...` a project file from either a file browser or PostgreSQL database
 * :guilabel:`Close` a project or revert it to its last saved state
-* :guilabel:`Save` a project in a :file:`.qgs` or :file:`.qgz` file format, either as file
+* :guilabel:`Save` a project in :file:`.qgs` or :file:`.qgz` file format, either as a file
   or within a PostgreSQL database
-* Export the project rendering to different formats from the map canvas or
+* Export the project maps to different formats from the map canvas or
```
Comment (tomchadwin) [inline source/docs/user_manual/introduction/qgis_gui.rst]:
```suggestion
* Export the map canvas to different formats or
```

Comment (havatv) [reply source/docs/user_manual/introduction/qgis_gui.rst]: Fine

File: source/docs/user_manual/introduction/qgis_gui.rst (hunk 3 above)
Comment (havatv) [reply source/docs/user_manual/introduction/qgis_gui.rst]: fine

**Comment by havatv** [comment]
Thanks, @tomchadwin !

**Pull request merged**